tests/end_to_end/test_account.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import pytest_asyncio
+
+from kakarot_scripts.utils.starknet import call
+
+TOTAL_SUPPLY = 10000 * 10**18
+TEST_AMOUNT = 10 * 10**18
+
+
+@pytest.fixture(scope="session")
+async def class_hashes():
+    """
+    All declared class hashes.
+    """
+    from kakarot_scripts.utils.starknet import get_declarations
+
+    return get_declarations()
+
+
+@pytest_asyncio.fixture(scope="function")
```

Comment:
`scope='function'` so you want to deploy a new token each time?

Reply:
no, once is enough

Reply:
actually - yes, as I want the deployer to be the new_account whose scope is "function" now.